server/src/database/dummy-data/index.js

```diff
@@ -0,0 +1,33 @@
+import dotenv from 'dotenv';
+import createUsers from './users';
+import createPatients from './patients';
+import createGeneralTherapy from './general-therapy';
+import createTeethTherapy from './teeth-therapy';
+import createPatientTeethTherapy from './patient-teeth-therapy';
+// import createPatientGeneralTherapy from './patient-general-therapy';
+import createFinancialRecords from './financial-records';
+import createAppointments from './appointments';
+
+if (process.env.NODE_ENV !== 'production') {
+  dotenv.config();
+}
+
+const buildData = async () => {
+  const createdData = {};
+  createdData.users = await createUsers(createdData);
```

Comment:
Why are you passing `createdData` as an argument, although the function does not take one?
And the same for the rest of the `create` functions.